enable/__init__.py

```diff
@@ -16,7 +16,7 @@
     __version__ = "not-built"
 
 __requires__ = [
-    "numpy", "pillow", "traits>=6.2.0", "traitsui", "pyface>=7.2.0", "fonttools"
```

Comment:
Do we still need this change to the Pyface requirement?

Reply:
Since many of the changes are depending on pyface 8.00 which renamed all qt4 to qt, I think we need pyface 8.00

Reply:
Let me check on that, whether any issues will be raised in pyface < 8

Reply:
Please could you point to some examples? I'm not seeing anything in this PR which would require us to use Pyface 8.0.

Reply:
@homosapien-lcy Ah, you edited your comment after I replied to it. Please don't do that! (Now my replies don't make sense out of context.)